swri_profiler_tools/src/profile.cpp

```diff
@@ -300,7 +300,7 @@ void Profile::rebuildFlatIndex()
 }
 
 // Compares the first N items of two string lists.
-static bool compareInitialStringList(
+bool compareInitialStringList(
```

Comment:
I'm not a fan of this change.  Why do you want to remove it?  Labeling a function static makes it very clear that it cannot be not called from anywhere outside this file, which is great for knowing the risk of changing it.